tests/unit/resource/resource_tests.hpp

```diff
@@ -13,6 +13,7 @@
 
 #include "umpire/util/Platform.hpp"
 #include "umpire/util/MemoryResourceTraits.hpp"
+#include "umpire/util/Macros.hpp"
```

Comment:
Why was this header file added?  I don't think it should be needed here.

Reply:
If we remove this line we get this error... 

`[ 49%] Building CXX object tests/unit/resource/CMakeFiles/file_resource_tests.dir/file_resource_tests.cpp.o
In file included from /home/aileen/Umpire/tests/unit/resource/file_resource_tests.cpp:7:
/home/aileen/Umpire/tests/unit/resource/resource_tests.hpp: In instantiation of ‘void gtest_suite_ResourceTest_::getTraits<gtest_TypeParam_>::TestBody() [with gtest_TypeParam_ = umpire::resource::FileMemoryResource]’:
/home/aileen/Umpire/tests/unit/resource/resource_tests.hpp:75:1:   required from here
/home/aileen/Umpire/tests/unit/resource/resource_tests.hpp:79:17: error: ‘UMPIRE_USE_VAR’ was not declared in this scope
   79 |   UMPIRE_USE_VAR(traits);
      |   ~~~~~~~~~~~~~~^~~~~~~~
/home/aileen/Umpire/tests/unit/resource/resource_tests.hpp:79:17: note: the macro ‘UMPIRE_USE_VAR’ had not yet been defined
In file included from /home/aileen/Umpire/src/umpire/util/MemoryMap.inl:10,
                 from /home/aileen/Umpire/src/umpire/util/MemoryMap.hpp:193,
                 from /home/aileen/Umpire/src/umpire/resource/FileMemoryResource.hpp:13,
                 from /home/aileen/Umpire/tests/unit/resource/file_resource_tests.cpp:9:
/home/aileen/Umpire/src/umpire/util/Macros.hpp:89: note: it was later defined here
   89 | #define UMPIRE_USE_VAR(x) static_cast<void>(x)
      | 
make[2]: *** [tests/unit/resource/CMakeFiles/file_resource_tests.dir/build.make:63: tests/unit/resource/CMakeFiles/file_resource_tests.dir/file_resource_tests.cpp.o] Error 1
make[1]: *** [CMakeFiles/Makefile2:2301: tests/unit/resource/CMakeFiles/file_resource_tests.dir/all] Error 2
make: *** [Makefile:141: all] Error 2
`

Reply:
Thanks @aileenperez, you are correct.